### PR TITLE
registration page

### PR DESCRIPTION
**Pull request by RastvorovAnton**

### CODE REVIEW

File: src/components/Pages/RegistrationPage.scss [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+.main-and-form {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	margin-top: 41px;
+}
+
+@media screen and (max-width: 890px) {
+	.Header header .textHeader {
```
Comment (NataliaShchelkunova) [inline src/components/Pages/RegistrationPage.scss]: . textHeader in new line

File: src/components/Pages/RegistrationPage.scss [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+	}
+	.all-cont {
+		width: 18em;
+
+		.RegistrationForm {
+			width: 20em;
+		}
+	}
+
+	.Header header .textHeader {
```
Comment (NataliaShchelkunova) [inline src/components/Pages/RegistrationPage.scss]: textHeader in new line

File: src/components/Pages/RegistrationPage.scss [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+	.LogoCont {
+		margin-left: 0;
+	}
+	.all-cont {
+		width: 18em;
+		.RegistrationForm {
+			width: auto;
+		}
+	}
+	.Header header .textHeader {
```
Comment (NataliaShchelkunova) [inline src/components/Pages/RegistrationPage.scss]: textHeader in new line

File: src/components/RegistrationPage/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+import image from '../../../icons/Vector.png'
+import '../Header/Header.scss'
+
+const Header = () => {
+
```
Comment (NataliaShchelkunova) [inline src/components/RegistrationPage/Header/Header.jsx]: remove empty line

File: src/components/RegistrationPage/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	let headerText = '';
+	let buttonExit = 
+
+		<Link to='/authorization'>
+			<button className='btn-exit'>Выход</button>
+		</Link>
```
Comment (NataliaShchelkunova) [inline src/components/RegistrationPage/Header/Header.jsx]: why is it here?

File: src/components/RegistrationPage/Header/Header.scss [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+		.textHeader {
+			display: flex;
+			align-items: center;
+			justify-content: flex-start;
+			font-family: "Roboto";
+			font-style: normal;
+			font-weight: 300;
+			font-size: 36px;
+			color: #000000;
+		}
```
Comment (NataliaShchelkunova) [inline src/components/RegistrationPage/Header/Header.scss]: add emptyline

File: src/components/RegistrationPage/MainContainer/RegistationForm/RegistrationForm.jsx [hunk 7]
```diff
@@ -0,0 +1,166 @@
+import React, { useState, useEffect } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
+import axios from "axios";
```
Comment (NataliaShchelkunova) [inline src/components/RegistrationPage/MainContainer/RegistationForm/RegistrationForm.jsx]: import style

File: src/components/RegistrationPage/MainContainer/RegistationForm/RegistrationForm.jsx [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+import './RegistrationForm.scss';
+
+const Alert = React.forwardRef(function Alert(props, ref) {
+  return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
```
Comment (NataliaShchelkunova) [inline src/components/RegistrationPage/MainContainer/RegistationForm/RegistrationForm.jsx]: code style. new attributes in new line

File: src/components/RegistrationPage/Header/Header.jsx (hunk 5 above)
Comment (RastvorovAnton) [reply src/components/RegistrationPage/Header/Header.jsx]: why not? what should I do with it?

File: src/components/RegistrationPage/MainContainer/RegistationForm/RegistrationForm.jsx [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+            </div>
+          </div>
+          <div className='reg-link'>
+            <button disabled={!formValid} className="Registr" >Зарегистрироваться</button>
+            <Link to='/authorization'>
+              <span className='ButtonStyle' onClick={() => setClick(1)} >Авторизоваться</span>
+            </Link>
+          </div>
+        </div>
+        : click === 121331231
```
Comment (NataliaShchelkunova) [inline src/components/RegistrationPage/MainContainer/RegistationForm/RegistrationForm.jsx]: remove

**Pull request merged**